lib/diff.js

```diff
@@ -144,11 +145,21 @@ const getChildren = diff => {
   // responsible for installing.
   const actualKids = allChildren(actual)
   const idealKids = allChildren(ideal)
+
+  if (ideal && ideal.hasShrinkwrap && !shrinkwrapInflated.has(ideal)) {
+    // coverage disabled because this is difficult to reach in a synthetic test
```

Comment:
i tried every combination i could think of to try to reach this line, and wasn't able to. @isaacs maybe if you see what i'm missing i can adjust the test cases and remove this

Reply:
Turns out that case is impossible to hit.  I traced it through and replaced the istanbul ignore and the `if` with a comment explaining why we're not being careful there.